metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_audit.py

```diff
@@ -22,24 +22,29 @@
 timestamp >= "{start_time}"
 AND
 timestamp < "{end_time}"
+AND protoPayload.serviceName="bigquery.googleapis.com"
 AND
 (
     (
-        protoPayload.methodName=
-            (
-                "google.cloud.bigquery.v2.JobService.Query"
-                OR
-                "google.cloud.bigquery.v2.JobService.InsertJob"
-            )
```

Comment:
Slightly worried about this change, since this is a big filter, but I assume the later ones catch it (e.g. if jobChange.job exists, then method name is query / insertjob)

Reply:
right, that's my understanding as well. Let me keep these filters if this is worrisome. I was simply trying to keep same filters across gcp logging client and exported audit logs.